tests/test_api_bitcast.py

```diff
@@ -0,0 +1,84 @@
+import heterocl as hcl 
+import numpy as np
+
+
+def test_bitcast_uint2float():
+    hcl.init()
+    A = hcl.placeholder((10,10), dtype=hcl.UInt(32), name='A')
+    
+    def algorithm(A):
+        B = hcl.bitcast(A, hcl.Float(32))
+        return B
+        
+
+    s = hcl.create_schedule([A], algorithm)
+    f = hcl.build(s)
+
+    print(hcl.lower(s))
```

Comment:
Remove all `print(hcl.lower(s))` in tests.

Reply:
Done